src/logic/embedding.lean

```diff
@@ -34,9 +34,18 @@ end function
 protected def equiv.to_embedding {α : Sort u} {β : Sort v} (f : α ≃ β) : α ↪ β :=
 ⟨f, f.injective⟩
 
+/-- Convert an `α ↬ β` to `α ↪ β`. -/
+@[simps]
+protected def injection.to_embedding {α : Sort u} {β : Sort v} (f : α ↬ β) : α ↪ β :=
+⟨f, f.injective⟩
+
 namespace function
 namespace embedding
 
+/-- Convert an `α ↪ β` to `α ↬ β` by using `function.inv_fun` as the left-inverse. -/
```

Comment:
Do you also want to provide a default computable left inverse for `fintype α` and `decidable_eq β`?

Reply:
Feel free to add one to this branch, assuming you already have the implementation.

Reply:
I feel like such a function should be provided by `fintype` somewhere, and be cross-referenced from `function.inv_fun`. Do you want to add that in a separate PR?